src/PostgREST/Request/ApiRequest.hs

```diff
@@ -18,11 +18,14 @@ module PostgREST.Request.ApiRequest
   ) where
 
 import qualified Data.Aeson            as JSON
+import qualified Data.Aeson.Key        as K
+import qualified Data.Aeson.KeyMap     as KM
 import qualified Data.ByteString.Char8 as BS
 import qualified Data.ByteString.Lazy  as LBS
 import qualified Data.CaseInsensitive  as CI
 import qualified Data.Csv              as CSV
 import qualified Data.HashMap.Strict   as M
+import qualified Data.Map.Strict       as MS
```

Comment:
I would have preferred calling this `M`, but renaming `HashMap` to `HM` causes linter problems with all the `M` imports elsewhere in the codebase.

Reply:
Had to rename after all to placate the linters...